src/foam/layout/Section.js

```diff
@@ -138,7 +138,7 @@ foam.CLASS({
         this.properties = a.properties.map(p => {
           if ( foam.String.isInstance(p) ) return cls.getAxiomByName(p);
           if ( p.name ) return cls.getAxiomByName(p.name).clone().copyFrom(p);
-        });
+        }).sort((p1, p2) => p1.order - p2.order);
```

Comment:
sort(foam.core.Property.ORDER.compare)
instead of
sort((p1, p2) => p1.order - p2.order)

It isn't any shorter, but works even if we change the type of order.

Reply:
thanks @kgrgreer  updated